web_ui/src/pages/camera-page/components/sidebar/device-settings.component.tsx

```diff
@@ -31,28 +54,38 @@ export const DeviceSettings = () => {
                 {({ deviceId, label }) => <Item key={deviceId}>{label}</Item>}
             </Picker>
 
-            <Divider size={'S'} marginTop={'size-250'} marginBottom={'size-250'} />
-
-            <SettingOption
-                label='Mirror camera'
-                config={{ type: 'selection', options: ['Off', 'On'], value: isMirrored ? 'On' : 'Off' }}
-                onChange={(value) => {
-                    setIsMirrored(value === 'On');
-                }}
-            />
-
-            {sortedByOptions.map(({ name, config }) => (
-                <SettingOption
-                    key={name}
-                    label={name}
-                    config={config}
-                    onChange={(value) => {
-                        if (webcamRef.current?.stream) {
-                            applySettings(webcamRef.current?.stream, { [name]: value });
-                        }
-                    }}
-                />
+            {categories.map(({ categoryName, attributesKeys }) => (
+                <Disclosure key={categoryName}>
```

Comment:
[Suggestion] Move this to a `category` component, making the render easier to read and making this part easier to test